### ISSUE TITLE
Height of box characters in "Symbols for Legacy Computing" doesn't match height of box characters in "Block Elements" block 

### ISSUE THREAD

**Issue by nico** [notofonts/symbols#26]
I was playing with the "Symbols for Legacy Computing" block a bit, and I noticed that https://www.unicode.org/charts/PDF/U1FB00.pdf mentions that "1FB88 🮈 RIGHT THREE EIGHTHS BLOCK" refers to "→ 258D ▍  left three eighths block" as the corresponding box on the other side.

As far as I can tell, these two glyphs have different heights in Noto. Is that intentional? It sounds like the "Block Elements" glyphs in U+258x should have the same height as the glyphs in "Symbols for Legacy Computing"

![screenshot](https://user-images.githubusercontent.com/3487/107821874-33904600-6d4b-11eb-8a7d-cf9b2f3f40d4.png)

(Same for the other box drawing characters in "Symbols for Legacy Computing" -- affects most glyphs in that block.)

**Comment by simoncozens** [comment]
> As far as I can tell, these two glyphs have different heights in Noto.

This is true, but these two blocks appear in two different Noto fonts, with different vertical metrics, so I am not convinced they would work together well even if properly aligned. I suspect the right thing to do here is to add all the Block Elements to symbols as well, in a way that's compatible with the Symbols For Legacy Computing.

**Comment by simoncozens** [comment]
(This is now the same as #70)

**Comment by Mercury13** [comment]
I’ll repeat my solution. Support 2580…9F, with the same character cell size. That’s probably enough.

**Comment by simoncozens** [comment]
Yes, that's what I just said here:

>  I suspect the right thing to do here is to add all the Block Elements to symbols as well, in a way that's compatible with the Symbols For Legacy Computing.
